### DISCUSSION TITLE
Tokopedia: Cara membuat filter untuk hapus kolom iklan di browser mobile

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#725] [Q&A]
saya tanya lagi gimana buat filter hapus kolom iklan di browser mobile?

**Comment by realodix** [comment]
Maksudnya bagaimana ya, untuk aplikasi AdGuard Mobile?
Bisa kasih contoh?

**Comment by 3xploiton3** [comment]
Saya pakai firefox android, addon ublock origin

Tampilan dibawah ini versi mobile,  kotak yg saya tandai pengin saya
hilangkan

Contoh
https://m.tokopedia.com/search?q=bajaka&st=product

![](https://i.imgur.com/BwG9z3Y.jpg)

**Comment by realodix** [comment]
Coba pakai ini deh
`##.unf-card:has(img[alt*="TopAds"])`

**Comment by 3xploiton3** [comment]
@realodix belum visa,  gak Ada perubahan

**Comment by realodix** [comment]
> @realodix belum visa, gak Ada perubahan

Kurang tau deh, Saya coba sih bisa

**Comment by realodix** [comment]
![Screenshot_2019-10-17-20-29-25-861_org mozilla firefox](https://user-images.githubusercontent.com/1314456/67013404-42e28d80-f11d-11e9-9253-dd80e9013dd5.png)

**Comment by 3xploiton3** [comment]
Saya edit akhirnya bisa, thx bro 👍

```
m.tokopedia.com##.unf-card:has(img[alt*="TopAds"]) 
```
